File: SecondoAnno/Python/esame_11_04_2019/dipendente.py
class Dipendente:
    codice: int 
    nome: str
    costo_orario: float
    servizi: list

    def __init__(self, codice: int, nome: str, costo_orario: float, servizi: list) -> None:
        self.codice = codice
        self.nome = nome
        self.costo_orario = costo_orario

        if servizi is None:
            self.servizi = []
        else:
            self.servizi = servizi

    def add_servizio(self, servizio):
        self.servizi.append(servizio)

    def __str__(self) -> str:
        return self.nome + " , " + str(self.codice) + " , "

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame11Aprile2019/Dipendente.java
public class Dipendente {

    protected int codice;
    protected String nomeDipendente;
    protected double costoOrario;

    public Dipendente(int codice, String nomeDipendente, double costoOrario) {
        this.codice = codice;
        this.nomeDipendente = nomeDipendente;
        this.costoOrario = costoOrario;
    }

    public int getCodice() {
        return codice;
    }

    public double getCosto() {
        return costoOrario;
    }

    public String getNome() {
        return nomeDipendente;
    }
}